deployments/liqo/values.yaml

```diff
@@ -189,6 +189,18 @@ nameOverride: ""
 # -- full liqo name override
 fullnameOverride: ""
 
+# aws configuration for the local cluster and the Liqo user
```

Comment:
Would add a couple of information about what capabilities the liqo user should have (with a pointer to the doc when it will be introduced)